doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -247,6 +247,19 @@ SQL objects that don't exist in the compute layer (such as views) are omitted.
 | `object_id`     | [`text`] | The ID of a compute object. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes), [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views), or [`mz_internal.mz_subscriptions`](#mz_subscriptions).                                                           |
 | `dependency_id` | [`text`] | The ID of a compute dependency. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes), [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views), [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources), or [`mz_catalog.mz_tables.id`](../mz_catalog#mz_tables). |
 
+### `mz_compute_hydration_status`
```

Comment:
Do you think once we figure out how to get this information for storage objects, we would replace this relation with `mz_hydration_status` that contains both types of objects?

Reply:
That depends on how we can get the information from storage objects. If we can get it in the controller, we can replace this relation with `mz_hydration_status`. But we may instead have to stream it from the replica through statistics `StorageResponses`, in which case it would end up in `mz_source_statistics` and duplicating it would perhaps not make much sense.